### PR TITLE
MueLu: Clean ups for matrix-free example 01, fix Belos route

### PR DESCRIPTION
**Pull request by GrahamBenHarper**
@trilinos/muelu @pohm01 

Just some minor cleanups with the matrix-free driver that I had sitting around, like fixing the non-multigrid route and adding some more multigrid settings.

### CODE REVIEW

File: packages/muelu/research/graham/mf_example_01.cpp [hunk 1]
```diff
@@ -528,85 +521,99 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib lib, int arg
       auto rhs_1d = Kokkos::subview (rhs_2d, Kokkos::ALL(), 0);
       SC x = x_left;
       for(size_t i=0; i<n_local; ++i) {
-        rhs_1d(i) = 4*h*h*PI*PI*sin(2*PI*x);
```
Comment (cgcgcg) [inline packages/muelu/research/graham/mf_example_01.cpp:531]: Could use the math constants from Kokkos..

**Review by cgcgcg — APPROVED** [review]

File: packages/muelu/research/graham/mf_example_01.cpp (hunk 1 above)
Comment (GrahamBenHarper) [reply packages/muelu/research/graham/mf_example_01.cpp:531]: I think when I first started this, some of those constants were in the Kokkos experimental namespace. Are they in a different namespace now?

Comment (cgcgcg) [reply packages/muelu/research/graham/mf_example_01.cpp:531]: https://kokkos.github.io/kokkos-core-wiki/API/core/numerics/mathematical-constants.html

Comment (GrahamBenHarper) [reply packages/muelu/research/graham/mf_example_01.cpp:531]: Perfect, thanks! I'll queue that up for next time :)

**Pull request merged**